packages/core/strapi/ee/index.js

```diff
@@ -165,6 +165,10 @@ module.exports = Object.freeze({
     return ee.enabled;
   },
 
+  get licenseInfo() {
+    return Object.freeze(ee.licenseInfo);
```

Comment:
This is not sufficient, `licenseInfo` can - and does - still contain non-primitive values, meaning that they still will be easily modifiable from outside the module cause the object is not deeply frozen.

For simplicity sake I'll just recommend to expose a getter for the `seats`.

Reply:
`ee.licenseInfo.seats`, aye